scripts/sct_download_data.py

```diff
@@ -92,6 +92,7 @@ def main(args=None):
         'course_hawaii17': 'https://osf.io/6exht/?action=download',
         'deepseg_gm_models': ['https://osf.io/y8k2e/?action=download',
                               'https://www.neuro.polymtl.ca/_media/downloads/sct/20171126_deepgmseg_models.zip'],
+        'deepseg_sc_models': 'https://osf.io/86phg/?action=download'
```

Comment:
The alternative mirror must be provided.

Reply:
It's already uploaded (see my email to you and Charley on Jan 25). URL is:
https://www.neuro.polymtl.ca/_media/downloads/sct/20180125_deepseg_sc_models.zip